packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -35,7 +37,7 @@ export interface DesignSystem {
     contrast: number;
 
     /**
-     * The density multiplier
+     * The density offset. Plus or minus 1 - 3.
      */
     density: number;

```

Comment:
should this be set to `densityOffset`? Also I'm finding the wording in the comment confusing, do you mean to say "The density offset which can be an integer in a range from -3 to 3"? Also you could more strongly type this by changing this to `densityOffset: -3 | -2 | -1 | 0 | 1 | 2 | 3;`

Reply:
Made a DensityOffset type.